api/api.go

```diff
@@ -78,6 +78,7 @@ func routes(host, secretKey string, router *mux.Router, dataStore store.DataStor
 	api.router.HandleFunc("/instances", api.privateAuth.Check(instance.Add)).Methods("POST")
 	api.router.HandleFunc("/instances/{id}", api.privateAuth.Check(instance.Get)).Methods("GET")
 	api.router.HandleFunc("/instances/{id}", api.privateAuth.Check(instance.Update)).Methods("PUT")
+	api.router.HandleFunc("/instances/{id}/dimensions/{dimension}", api.privateAuth.Check(instance.UpdateDimension)).Methods("PUT")
```

Comment:
If this is a new API route then we also want to update the swagger spec as well.